src/mapper.js

```diff
@@ -56,6 +56,31 @@ const getSet = (mapItems, properties) => {
   }
 };
 
+const getPrimitivesSet = (sourceValues, paths) => {
+  if(Array.isArray(paths)) {
+    let resolvedValues = [];
+    if(Array.isArray(sourceValues)) {
+      sourceValues.forEach(sourceValue => {
+        for (const item in paths) {
+          if(paths[item]) {
+            if(get(sourceValue, paths[item])) {
+              resolvedValues.push(get(sourceValue, paths[item]));
+              break;
+            }
+          } else {
+            if(typeof sourceValue !== "object") {
```

Comment:
https://www.npmjs.com/package/is-primitive
```suggestion
            const isPrimitive = require('is-primitive');
            if(isPrimitive(sourceValue)) {
```